pages/common/flyctl.md

```diff
@@ -0,0 +1,36 @@
+# flyctl
+
+> Command-line tool for flyctl.io.
+> More information: <https://github.com/superfly/flyctl>.
+
+- Sign into fly account:
+
+`flyctl auth login`
+
+- Launch from your own Dockerfile:
+
+`flyctl launch`
```

Comment:
Although the default is the `Dockerfile` in the current directory, how about we still specify it explicitly:
```suggestion
`flyctl launch --dockerfile {{path/to/Dockerfile}}`
```